config_github_check_run.go

```diff
@@ -0,0 +1,55 @@
+package evergreen
+
+import (
+	"context"
+
+	"github.com/pkg/errors"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+)
+
+// GitHubCheckRunConfig configures settings for the GitHub Checkrun API.
+type GitHubCheckRunConfig struct {
+	// CheckRunLimit is the number of check runs that Evergreen is willing to support per GitHub patch.
```

Comment:
Can we disambiguate what patches we're referring to by "GitHub patch"? Does this mean that it applies to GitHub merge queue patches, or just GitHub PRs?